src/Components/OATPropertyEditor/PropertiesModelSummary.tsx

```diff
@@ -40,41 +40,84 @@ export const PropertiesModelSummary = ({
     const iconWrapStyles = geIconWrapFitContentStyles();
     const generalPropertiesWrapStyles = getGeneralPropertiesWrapStyles();
     const textFieldStyes = getPropertyEditorTextFieldStyles();
-    const [nameLengthError, setNameLengthError] = useState(false);
+    const [displayNameLengthError, setDisplayNameLengthError] = useState(null);
+    const [idLengthError, setIdLengthError] = useState(null);
+    const { model, models } = state;
+    const [
+        errorDisplayNameAlreadyUsed,
+        setErrorDisplayNameAlreadyUsed
+    ] = useState(null);
+    const [errorIdAlreadyUsed, setErrorIdAlreadyUsed] = useState(null);
+    const [
+        errorRepeatedRelationshipName,
+        setErrorRepeatedRelationshipName
+    ] = useState(null);
+    const [errorNameAlreadyUsed, setErrorNameAlreadyUsed] = useState(null);
+    const [nameLengthError, setNameLengthError] = useState(null);
     const [nameValidCharactersError, setNameValidCharactersError] = useState(
         false
     );
-    const [displayNameError, setDisplayNameError] = useState(null);
-    const [idLengthError, setIdLengthError] = useState(null);
     const [idValidDTMIError, setIdValidDTMIError] = useState(null);
-    const { model } = state;
-
-    const handleDisplayNameChange = (value) => {
-        if (value.length <= OATDisplayNameLengthLimit) {
-            const modelCopy = deepCopy(model);
-            modelCopy.displayName = value;
-            dispatch({
-                type: SET_OAT_PROPERTY_EDITOR_MODEL,
-                payload: modelCopy
-            });
+    const [displayName, setDisplayName] = useState(
+        model && model.displayName ? model.displayName : ''
+    );
+    const [name, setName] = useState(model && model.name ? model.name : '');
+    const [id, setId] = useState(model && model['@id'] ? model['@id'] : '');
 
-            setDisplayNameError(null);
-        } else {
-            setDisplayNameError(true);
+    useEffect(() => {
+        if (model) {
+            setDisplayName(model.displayName);
```

Comment:
What happens if this display name is localized? Do we need to do something different here or will it figure it out? 🤔